src/main/webapp/app/shared/image-cropper/component/image-cropper.component.ts

```diff
@@ -156,7 +157,9 @@ export class ImageCropperComponent implements OnChanges, OnInit {
             this.reset();
         }
         if (changes.imageChangedEvent && this.isValidImageChangedEvent()) {
-            this.loadImageFile(this.imageChangedEvent.target.files[0]);
+            const element = this.imageChangedEvent?.currentTarget as HTMLInputElement;
+            // @ts-ignore Ignore "is possibly null", checked before
```

Comment:
Why don't you simply add the check in the condition again?
I don't see an advantage of using a comment instead

Reply:
Not sure what you mean by "again". There was no null-check necessary before because there was no need for casting. Now there is a need and ?/! doesn't work for null values so we either have to do an if clause (O(1) and 2 lines) or ignore it (O(0) and 1 line)

Reply:
I'd simply add the check into the existing condition, so you would actually save one line. The runtime of a simple null check is negligible and should not be a reason for leaving it out.
Additionally it's better to be safe and check something again, since some future refactoring might break it.

Reply:
How would you want to do that? We need the null check on the cast, not before. Unless would want to do two casts. Or do you want me to pull the cast out of the conditional?

Reply:
Ok, you need an additional line...
I'd personally still prefer it over ignoring the linter

Reply:
Replaced it. Actually it's the `files` property that is nullable.